pulsar-client-cpp/lib/PartitionedConsumerImpl.h

```diff
@@ -60,6 +60,7 @@ namespace pulsar {
         virtual void redeliverUnacknowledgedMessages();
         virtual const std::string& getName() const;
         virtual int getNumOfPrefetchedMessages() const ;
+        virtual Result getConsumerStats(BrokerSideConsumerStats& brokerSideConsumerStats, int partitionIndex);
```

Comment:
As discussed yesterday - We can the code is not difficult but the interface will become quite ugly since now we need to return a list of stats always.